mod-audit-server/src/main/java/org/folio/verticle/acquisition/consumers/OrderEventsHandler.java

```diff
@@ -1,15 +1,51 @@
 package org.folio.verticle.acquisition.consumers;
 
 import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
 import io.vertx.kafka.client.consumer.KafkaConsumerRecord;
+import io.vertx.kafka.client.producer.KafkaHeader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.folio.kafka.AsyncRecordHandler;
+import org.folio.kafka.KafkaHeaderUtils;
+import org.folio.rest.jaxrs.model.Event;
+import org.folio.rest.jaxrs.model.OrderAuditEvent;
+import org.folio.rest.util.OkapiConnectionParams;
+import org.folio.services.OrderAuditEventService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import java.util.List;
+
 @Component
 public class OrderEventsHandler implements AsyncRecordHandler<String, String> {
 
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  public static final String RECORD_ID_HEADER = "recordId";
```

Comment:
Producer will not populate this header in our implementation, so we can delete it here

Reply:
fixed